src/server/models/userModel.ts

```diff
@@ -11,21 +12,21 @@ interface LoginDetails {
 }
 
 class UserModel {
-  private cachedLogins: {[id: string]: LoginDetails};
+  async getLoginDetails(login: string): Promise<LoginDetails|null> {
```

Comment:
It was nice in your subscriptions PR that you had a block comment describing the database layout. Can you do the same here somewhere?